publish.sh

```diff
@@ -41,15 +41,22 @@ fi
 
 if [[ $RELEASE_TYPE = 'final' || $RELEASE_TYPE = 'candidate' ]]
 then
-  # create the tag once, then cross-compile
+  # use our default scala version (2.12). compatible with all modules
   echo "creating repo tag for $RELEASE_TYPE release"
   ./gradlew -Prelease.scope=$RELEASE_SCOPE clean $RELEASE_TYPE
+
+  # then publish our primary module with other scala versions
   echo "publishing $REPOSITORY artifacts for $RELEASE_SCOPE $RELEASE_TYPE release"
-  ./gradlew -Prelease.useLastTag=true $PUBLISH_TASK
+  ./gradlew -Prelease.useLastTag=true -PscalaVersions=2.11.11 :warp-core:$PUBLISH_TASK
```

Comment:
maybe we can grep/read in the `scalaVersions` from `gradle.properties` here? Not a big deal since we won't be changing it often

Reply:
yeah, i'd say its a bit suboptimal but not a big deal. i have one other idea to try, dynamically disabling all gradle tasks for warp-core-gatling when scala version is not a 2.12